### PR TITLE
Added parse for keyvalued map(struct)

### PR DESCRIPTION
**Pull request by joekgu**
Cloning and modifying BigQueryUtils from `import org.apache.beam.sdk.io.gcp.bigquery.BigQueryUtils;` and adding it as BQUtils in our project.

The big change is going from: 

JSON -> Beam Row

TO:

 JSON -> Bigquery TableRow -> Beam Row

TableRow tr = convertJsonToTableRow(json.toString());
Row row = BqUtils.toBeamRow(schema, tr);

In order for this to work we had to modify a number of things in the BigQueryUtils package. I will try to make a push for them to accept these changes 😃

### CODE REVIEW

File: core/src/main/java/org/streamprocessor/core/transforms/SerializeMessageToRowFn.java [hunk 1]
```diff
@@ -150,14 +155,27 @@ public SerializeMessageToRowFn(
         this.ratio = 0.001f;
     }
 
-    @Setup
-    public void setup() throws Exception {}
+    public static TableRow convertJsonToTableRow(String json) {
```
Comment (ML-engineer) [inline core/src/main/java/org/streamprocessor/core/transforms/SerializeMessageToRowFn.java:158]: is this the same as the jsontotablerow.java in utils folder? Maybe use that instead instead?

File: core/src/main/java/org/streamprocessor/core/utils/SchemaUtils.java [hunk 2]
```diff
@@ -93,6 +103,18 @@ private static Field toBeamField(ColumnSchema column) {
         return field;
     }
 
+    private static boolean isKeyValueMap(ColumnSchema column) {
```
Comment (ML-engineer) [inline core/src/main/java/org/streamprocessor/core/utils/SchemaUtils.java:106]: Nice!

File: core/src/main/java/org/streamprocessor/core/utils/SchemaUtils.java [hunk 3]
```diff
@@ -101,7 +123,8 @@ private static FieldType getBeamFieldType(ColumnSchema column) {
         }
 
         if ("STRUCT".equals(dcFieldType)) {
-            Schema structSchema = fromColumnsList(column.getSubcolumnsList());
+            List<ColumnSchema> subColumns = column.getSubcolumnsList();
+            Schema structSchema = fromColumnsList(subColumns);
```
Comment (ML-engineer) [inline core/src/main/java/org/streamprocessor/core/utils/SchemaUtils.java:127]: how is this different?

**Review by ML-engineer — APPROVED** [review]
AWESOME! Great work Johan, hope you enjoyed parts of it even if it was a tricky one :)

File: core/src/main/java/org/streamprocessor/core/utils/SchemaUtils.java (hunk 3 above)
Comment (joekgu) [reply core/src/main/java/org/streamprocessor/core/utils/SchemaUtils.java:127]: Probably changed it for debugging, will change it back!

**Pull request merged**